src/core/public/index.scss

```diff
@@ -2,6 +2,5 @@
 @import './mixins';
 @import './core';
 @import './chrome/index';
-@import './overlays/index';
```

Comment:
This is likely the change that is breaking all the tests. 

@elastic/kibana-operations How can we import a scss file from a package from our core scss entrypoint? (the `./overlays/index` scss file was moved to a package)

Reply:
> This is likely the change that is breaking all the tests.

Yes, removing the styles was the culprit!

Reply:
For now, I've gone with _duplicating_ the sass in a new `overlays.scss` file while retaining the styles in the `core-overlays-browser-internal` package. The pragmatic approach would be to declare stylesheets in a single place (core for now), I'm not sure how the tooling will evolve, and fear we'll lose the reference to services that declare these classes.

Reply:
For now, I've gone with duplicating the sass in core/public while also keeping the styles in the core-overlays-browser-internal package, knowing full well that the pragmatic approach would be to declare stylesheets in a single place.
However, core and core's packages depend on the overlays packages directly, so we cannot remove the styles from the package itself either.

Reply:
> so we cannot remove the styles from the package itself either.

Not sure to get that?

Reply:
It's more of a hypothetical case if a service depends on the overlays as packages directly and doesn't use legacy core at all.